app/script_variant.py

```diff
@@ -145,6 +145,53 @@ def romanize_word_by_unidecode(word):
     romanized_word = unidecode(word)
     return [romanized_word.strip()]
 
+def is_chinese_person_name(word):
+    """
+    This method checks if a unicode string is a chinese person name by figuring out
+    if it contains valid a chinese surname and a given name.
+    
+    Return:
+        Boolean
+    """
+    surname, lastname  = split_chinese_name(word)
+    return surname is not None and lastname is not None
+
+def split_chinese_name(word):
+    """
+    This method tries to split a chinese name into two parts: surname and lastname
+    Args:
+        word: a chinese name string
+    Returns:
+        surname, lastname if it is a valid chinese name
+        else None, None
+    """
+    word = word.replace(" ", "")
+    if len(word) != len(re.findall(ur"[\u4e00-\u9fa5]", word)):
+        return None, None
+
+    if word[:2] in CHINESE_FAMILY_NAME_DICTIONARY:
+        return word[:2], word[2:]
+
+    elif word[0] in CHINESE_FAMILY_NAME_DICTIONARY:
+        return word[0], word[1:]
+
+    return None, None
+
+def romanize_chinese_person_name(word):
```

Comment:
I suppose `get_person_ids_from_results` is used to handle location/name matching problem right. In this sense, I suppose that we don't have to modify the `get_person_ids_from_results` as long as the query will be romanized by `script_variant.romanize_search_query` in [make_or_regexp](https://github.com/google/personfinder/blob/bcec681718ae454ced01d00fe839026b18ab80ad/app/full_text_search.py#L32). what do you think?